### DISCUSSION TITLE
Be able to save choice for "The following files are newer on disk" window

### DISCUSSION THREAD

**Discussion by hwabis** [godotengine/godot-proposals#7355] [Editor]
I'm editing C# scripts in Visual Studio, and whenever I save in it, I have to click in Godot "Reload" when "The following files are newer on disk" pops up. I'd like the option to have it automatically reload every time.

I believe VS already has this feature (because when I make changes to the files in Godot, VS will automatically reload them).

This could be implemented with a checkbox in the popup, "Remember my choice" or something.

**Comment by theraot** [comment]
Sounds like a regression. A search in the issues takes me here: https://github.com/godotengine/godot/issues/72825
